extensions/qute/deployment/src/test/java/io/quarkus/qute/deployment/EscapingTest.java

```diff
@@ -26,11 +26,13 @@
                     .addAsResource(new StringAsset("{text} {other} {text.raw} {text.safe} {item.foo}"),
                             "templates/foo.html")
                     .addAsResource(new StringAsset("{item} {item.raw}"),
-                            "templates/item.html")
+                            "templates/item.xhtml")
                     .addAsResource(new StringAsset("{text} {other} {text.raw} {text.safe} {item.foo}"),
                             "templates/bar.txt")
                     .addAsResource(new StringAsset("{@java.lang.String text}{text} {text.raw} {text.safe}"),
-                            "templates/validation.html"));
+                            "templates/validation.html"))
+            .overrideConfigKey("quarkus.qute.content-types.xhtml", "application/xhtml+xml")
```

Comment:
Didn't you add this one to the default value? If so, are you really testing your config?

Reply:
> Didn't you add this one to the default value?

Not really. This one maps `xhtml` files to `application/xhtml+xml`. It's a different config property.